app/src/main/java/com/github/se/cyrcle/ui/map/MapConfig.kt

```diff
@@ -33,4 +41,82 @@ object MapConfig {
   fun DefaultStyle() {
     return MapStyle("mapbox://styles/seanprz/cm27wh9ff00jl01r21jz3hcb1")
   }
+
+  // Useful function to debug the downloaded tiles, can be called from mapScreen if needed
+  fun getAllTilesDownloaded() {
+    val tileStore = TileStore.create()
+    // Get a list of tile regions that are currently available.
+    tileStore.getAllTileRegions { expected ->
+      if (expected.isValue) {
+        expected.value?.let { tileRegionList ->
+          Log.d("MapScreen", "Existing tile regions: $tileRegionList")
+        }
+
+        expected.error?.let { tileRegionError ->
+          Log.e("MapScreen", "TileRegionError: $tileRegionError")
+        }
+      }
+    }
+  }
+
+  /**
+   * Download the tiles to local storage for a specific zone
+   *
+   * @param zone the zone to download
+   */
+  fun downloadZone(zone: Zone) {
+    // Defines the area to download
+    val geometry =
+        Polygon.fromLngLats(
+            listOf(
+                listOf(
+                    zone.boundingBox.southwest(),
+                    zone.boundingBox.northeast(),
+                )))
+
+    // Define style and zooms levels to download
+    val tilesetDescriptor =
+        OfflineManager()
+            .createTilesetDescriptor(
+                TilesetDescriptorOptions.Builder()
+                    .styleURI("mapbox://styles/seanprz/cm27wh9ff00jl01r21jz3hcb1")
```

Comment:
Maybe we could keep this string in a constant somewhere? I is starting to get duplicated

Reply:
Done !